src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -31,11 +39,21 @@ var (
 
 func init() {
 	lang, _ = jibber_jabber.DetectLanguage()
-	country, _ = jibber_jabber.DetectTerritory()
 	tz = time.Local.String()
 }
 
-func configureLogging() *rotator.SizeRotator {
+func Setup(version string, buildDate string) *rotator.SizeRotator {
```

Comment:
`Setup` is mean to be to set `version`, `buildDate`, which are values that won't change and to initialize error writers and the loggly client. While `Configure` expects to receive a `config.Config` struct, which we can expect to change frequently and it only updates the HTTP client the client that was already created in `Setup`.